test/Xpring/xpring-client.test.ts

```diff
@@ -6,7 +6,6 @@ import FakeXrpPayIdClient from '../PayID/fakes/fake-xrp-pay-id-client'
 import { testXrpTransaction } from '../XRP/fakes/fake-xrp-protobufs'
 import TransactionStatus from '../../src/XRP/transaction-status'
 import XpringClient from '../../src/Xpring/xpring-client'
-import RawTransactionStatus from '../../src/XRP/raw-transaction-status'
```

Comment:
again, just seeing the repercussions of being able to remove some methods from `XrpClientDecorator` and all its implementing classes, and sticking them in one place, the `CommonXrplClient`